Diplomas/DiplomaSetupTest.php

```
@@ -0,0 +1,165 @@
+
+        $config = \pl_diploma_get_config(7);
+
+        $this->assertSame('Mistrzostwa Polski', $config['CompetitionName']);
+        $this->assertSame(1, $config['TitlesEnabled']);
+    }
+
+    public function testSaveConfigInsertsWhenNotExists(): void
+    {
+        \pl_diploma_save_config(7, $this->sampleConfigData());
+
+        $this->assertCount(1, \FakeDb::executed('/INSERT INTO PLDiplomaConfig/'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\FakeDb' in method 'testSaveConfigInsertsWhenNotExists'. (undefined)

(StaticAccess)